src/main/scala/Adapter.scala

```diff
@@ -14,6 +14,8 @@ import rocketchip._
 import rocket.XLen
 import cde.{Parameters, Field}
 
+import chisel3.core.ExplicitCompileOptions.NotStrict
```

Comment:
Oh, I see. Is it true that the Chisel compatibility layer doesn't call IO()? The io is being instantiated by a base class in rocket.